third_party/proton/dialect/include/Conversion/ProtonGPUToLLVM/PatternProtonGPUOpToLLVM.h

```diff
@@ -6,6 +6,10 @@
 
 namespace mlir::triton {
 namespace proton::gpu {
+
+// Profiler index is private to each thread, address space is 5
+constexpr int indexPtrAddressSpace = 5;
```

Comment:
```suggestion
constexpr int IndexPtrAddrSpace = 5;
```
Constants might be better beginning with a capitalized letter.